### PR TITLE
Add section on Atlas - Unfinished

### PR DESCRIPTION
**Pull request by Alonski**

### CODE REVIEW

**Comment by amyrlam** [comment]
hey alon, thanks for your PR! i can assist in taking this over the finish line

File: source/2020-01-10-the-ember-times-issue-130.md [hunk 1]
```diff
@@ -70,9 +71,17 @@ If you need help along the way, visit the [Ember Community chat and forums](http
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [The Ember Atlas 🗺](https://www.notion.so/The-Ember-Atlas-4094f81c86c34badb4a562ed29414ae1)
```
Comment (amyrlam) [inline source/2020-01-10-the-ember-times-issue-130.md]:
```suggestion
## [The Ember Atlas 🗺](http://emberatlas.com)
```

I noticed that https doesn't work, but it's an easier to remember redirect

**Comment by amyrlam** [comment]
@Alonski i'm going to merge this in to get upstream up to date, and can finish it off. there is mention of atlas in another place so can see what works best

**Comment by Alonski** [comment]
Thanks @amyrlam !

**Pull request merged**